.buildkite/basic/browser-pipeline.yml

```diff
@@ -171,3 +168,32 @@ steps:
         concurrency: 2
         concurrency_group: "browserstack"
         concurrency_method: eager
+
+      #
+      # BrowserStack non-https tests
+      #
```

Comment:
not sure if there's a DRYer way of doing this rather than duplicating most of the previous section? A parameterized matrix, perhaps?

Reply:
I don't think so unfortunately, the Buildkite YAML schema is quite limited when it comes to any kind of commonality or logic between steps.